adapters/teads/teads.go

```diff
@@ -196,7 +196,7 @@ func getMediaTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType,
 			return openrtb_ext.BidTypeBanner, nil
 		}
 	}
-	return openrtb_ext.BidTypeBanner, []error{&errortypes.BadInput{
+	return openrtb_ext.BidType(""), []error{&errortypes.BadInput{
```

Comment:
As per https://github.com/prebid/prebid-server/pull/3112#issuecomment-1752931342,

This code path is not tested. Consider adding json test to cover this code path